spec/services/flow/steps/partner/full_employment_details_step_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe Flow::Steps::Partner::FullEmploymentDetailsStep, type: :request do
+  let(:legal_aid_application) { create(:legal_aid_application) }
+
+  describe "#path" do
+    subject { described_class.path.call(legal_aid_application) }
+
+    it { is_expected.to eql providers_legal_aid_application_partners_full_employment_details_path(legal_aid_application) }
+  end
+
+  describe "#forward" do
```

Comment:
Did this step not have any controller redirect tests?

Reply:
This should now be resolved.